src/commands/create-fixture.ts

```diff
@@ -53,7 +53,7 @@ export default async function main(directory: string, overwriteInfo: boolean) {
     writeJsonSync(resultFixturePath, actions);
 
     const mutationsFixturePath = join(fixturePath, "mutations.json");
-    const mutations = await executePrActions(actions, prInfo, /*dry*/ true);
+    const mutations = await executePrActions(response.data.repository!.id, actions, prInfo, /*dry*/ true);
     writeJsonSync(mutationsFixturePath, mutations);
```

Comment:
The API for re-running requires an ID to the repo, and that's not in the prInfo. Given it's only used here, I figure it made more sense to add it as an arg